### PR TITLE
Feature/14/14 exam functions/1 star triangle

### PR DESCRIPTION
**Pull request by sergeroshe**
Initial commit of star_triangle file

### CODE REVIEW

File: star_triangle.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+HEIGHT = 8
+FILL_SYM = ' '
+OUT_TEMPLATE = '*'
+BORDER_WIDTH = len(OUT_TEMPLATE)
+
+
+def draw_triangle():
+    half_base = BASE // 2
+    for i in range(HEIGHT):
+        print((FILL_SYM * (half_base * BORDER_WIDTH - i * BORDER_WIDTH)) + OUT_TEMPLATE * (i + 1) + OUT_TEMPLATE * i)
```
Comment (YanumVlad) [inline star_triangle.py:11]: 1. Violation of logic / output  rule:
extract calc to the separate var

Comment (YanumVlad) [inline star_triangle.py:11]: Simplify expression (math based):
a * b + a * c = a * (b + c)

2 * 3 + 2 * 5 = 2 * (3 + 5)

**Review by YanumVlad — DISMISSED** [review]

**Comment by YanumVlad** [comment]
@sergeroshe , if you want to conduct my review - change status of PR to Open. Now it is in Draft status

File: star_triangle.py (hunk 1 above)
Comment (sergeroshe) [reply star_triangle.py:11]: I don't understand how to perform it, please help me with this.

Comment (sergeroshe) [reply star_triangle.py:11]: This one I understand, thanks

**Comment by sergeroshe** [comment]
'if you want to conduct my review' not clear English, explain please

File: Звездный треугольник 🌶️.py [hunk 2]
```diff
@@ -5,14 +5,15 @@
 BORDER_WIDTH = len(OUT_TEMPLATE)
 
 
-def draw_triangle():
-    half_base = BASE // 2
+def draw_triangle(half_base):
```
Comment (YanumVlad) [inline Звездный треугольник 🌶️.py:8]: Твоя функция по прежнему еще зависит от констант

Comment (sergeroshe) [reply Звездный треугольник 🌶️.py]: После последнего коммита, there's still a mistake?

**Review by YanumVlad — APPROVED** [review]

**Pull request merged**